roscala/src/test/scala/coop/rchain/roscala/VmSpec.scala

```diff
@@ -3,16 +3,16 @@ package coop.rchain.roscala
 import coop.rchain.roscala.CtxtRegName._
 import coop.rchain.roscala.VmLiteralName._
 import coop.rchain.roscala.ob._
-import coop.rchain.roscala.ob.expr.TupleExpr
+import coop.rchain.roscala.ob.expr.{RequestExpr, TupleExpr}
 import coop.rchain.roscala.prim.fixnum.fxPlus
 import coop.rchain.roscala.prim.rblfloat.flPlus
 import org.scalatest.{FlatSpec, Matchers}
 
-import scala.collection.mutable
-
 class VmSpec extends FlatSpec with Matchers {
 
   val globalEnv = new GlobalEnv()
+  // this is defined in BigBang.cc#InitBuiltinObs
+  val topMeta = Meta.empty
```

Comment:
Please remove since we don't need that for now.

Reply:
Done.